java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -211,7 +211,7 @@ abstract class AbstractArcHost(
         spec: Plan.Particle,
         context: ArcHostContext
     ): ParticleContext {
-        val particle = instantiateParticle(ParticleIdentifier.from(spec.location))
+        val particle = instantiateParticle(ParticleIdentifier.from(spec.location), spec)
```

Comment:
At some point we should have a test that is using this capability. I understand that it may be premature now, as your next step is to explore how to use the Plan.Particle in the particle at runtime to do some interesting things - but maybe let's keep in mind that one you have a good example of that we could use that example as an integration test for this capability.

Reply:
The time for testing could very well be now! I can work on adding an integration test. I know that tests for this area are very much in flight anyway.